Assets/BossRoom/Scripts/Client/ClientDoorVisualization.cs

```diff
@@ -27,7 +27,7 @@ private void Awake()
         m_DoorState = GetComponent<NetworkDoorState>();
     }
 
-    public override void NetworkStart()
+    public override void OnNetworkSpawn()
```

Comment:
As discussed, let's also use the OnNetworkDespawn instead of our Destroys

Reply:
Done.